src/test/java/com/google/sps/ImageSelectionTest.java

```diff
@@ -0,0 +1,75 @@
+package com.google.sps.image;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.io.IOException;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public final class ImageSelectionTest {
+
+  private final Set<String> keywords = new LinkedHashSet<>();
+
+  // This is how every Bing image link starts
+  private static final String EXPECTED_LINK = "https://tse";
+
+  private void checkScrapingOutputLink() {
```

Comment:
The name of this method doesn't tell me what the scraped output link has to satisfy?

I should not have to read the code.